mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/course/LessonContentLesson.java

```diff
@@ -0,0 +1,43 @@
+package org.motechproject.nms.mobileacademy.domain.course;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+/**
+ * Contains details about actual content files to be played while playing a lesson.
+ */
+@Entity
+public class LessonContentLesson {
+
```

Comment:
Trying to keep up with the naming/nesting structure that Aricent came up with :( . I really want to fix this too but it would be harder to map what goes where with their schema if I diverge